### PR TITLE
Add two options for the CLI - SSL and DB auth

### PR DESCRIPTION
**Pull request by vaskort**
My PR is an:
💅 Enhancement 

Main update on the:
Framework

I wanted to dockerize my Strapi instance and realised that https://github.com/strapi/strapi-docker uses the CLI of Strapi to create a new Strapi intance (https://github.com/strapi/strapi-docker/blob/master/strapi.sh#L22). 

Added two new options for the CLI (`dbssl` and `dbauth`) to make it possible to connect to an Azure Cosmos DB database.

*Note*: I use the same options and credentials when not using the CLI to connect to my database and it connects successfully but with this way it is still timing out. 

Please take a look and let me know what you think I'm doing wrong or/and I'll refactor the code according your preferences.

### CODE REVIEW

**Comment by Aurelsicoko** [comment]
@vaskort It seems perfect to me. I also need to approval of @lucaperret on this one to make sure it won't have any impact with Docker

**Comment by lucaperret** [comment]
Hi @vaskort, thanks for this PR!
Could you also update the documentation ?

**Comment by vaskort** [comment]
Hi guys thanks for your comments!
@lucaperret Sure Just want to make sure it works - it might be something wrong with my database. 
Will come with an update later today!

File: docs/3.x.x/en/cli/CLI.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Create a new project
 ```bash
 strapi new <name>
 
-options: [--dev|--dbclient=<dbclient> --dbhost=<dbhost> --dbport=<dbport> --dbname=<dbname> --dbusername=<dbusername> --dbpassword=<dbpassword>]
+options: [--dev|--dbclient=<dbclient> --dbhost=<dbhost> --dbport=<dbport> --dbname=<dbname> --dbusername=<dbusername> --dbpassword=<dbpassword> --dbssl=<dbssl> --dbauth=<dbauth>]
```
Comment (vaskort) [inline docs/3.x.x/en/cli/CLI.md]: Do you think a better notation would be `--dbssl=[dbssl] --dbauth=[dbauth]`? Or should I leave it like this for consistency?

File: packages/strapi-generate-new/lib/before.js [hunk 2]
```diff
@@ -192,7 +192,7 @@ module.exports = (scope, cb) => {
               ])
               .then(answers => {
                 if (hasDatabaseConfig) {
-                  answers = _.omit(scope.database.settings, ['client']);
+                  answers = _.merge((_.omit(scope.database.settings, ['client'])), scope.database.options);
```
Comment (vaskort) [inline packages/strapi-generate-new/lib/before.js]: It seems I had to merge the optional options to the final `answers` object. Let me know if this makes sense.

**Comment by vaskort** [comment]
Hey guys,

It seems I had to merge the new optional options to the `answers` object (check my comment). Also thought to remove the new options from the `dbArguments` array for backward compatibility.

The docs have been updated.

Let me know your thoughts!

**Comment by Aurelsicoko** [comment]
@lucaperret Can you take the lead on this one? It's clearly out of my scope 🙈

**Pull request merged**